llvm/test/Transforms/LoopVectorize/AArch64/partial-reduce-dot-product-epilogue.ll

```diff
@@ -0,0 +1,161 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=1 -force-target-instruction-cost=1 -S < %s | FileCheck %s --check-prefix=CHECK-NODOTPROD
```

Comment:
If this is just for testing the TTI logic, is it sufficient to have a single test rather than adding check lines for all tests?

Reply:
Yes, IIUC the new `CHECK-NODOTPROD` is there to guard against the issue @david-arm pointed out with TTI. For that, a single test would probably be sufficient (in a separate file). And then remove the line from the other tests again.

Ideally also remove `-force-target-instruction-cost` if possible now that we have a separate file for fix-width tests?